### ISSUE TITLE
February release todo's

### ISSUE THREAD

**Issue by zfisch** [BRCAChallenge/brca-exchange#1008]
@melissacline  @amycoffin :
(please feel free to submit PRs for each issue or forward relevant content to @zfisch and he can submit PRs for your review)

- [x] update release notes for 2/14/19 data (first release with FA)
- [x] update disclaimer for FA tile
- [x] update top level readme of data download for FA (and other sources?)
- [x] update literature results beta disclaimer
- [x] send final data dictionary to zack for inclusion in data download

@falquaddoomi :

Submit PRs for
- [x] Data deployment updates
- [x] Printable variant details page
- [x] Google Analytics

@zfisch :

- [x] make functional assay tweaks in #1000 
- [x] submit PR to update LOVD retrieval
- [x] submit PR to include data dictionary in release
- [x] submit PRs as needed for any documentation updates
- [x] review and merge all PRs
- [x] regenerate 2/14/19 release with new release notes, data dictionary, and top level readme
- [x] use existing lit search data for this release

**Comment by amycoffin** [comment]
@zfisch feel free to change the first sentence of the LitSearch docs:

Current: "Literature Search is a beta feature, so please beware of erroneous or missing results."

Update: "Literature Search is a beta feature, so please beware of erroneous or missing results. To report this error, contact us." (Make contact us the same contact us link used elsewhere.)

**Comment by melissacline** [comment]
Text for release notes:

This release incorporates functional assay scores by FIndlay et al (PMID: 30209399).  This assay applies Saturation Genome Editing and a haploid cell line to derive a functional readout of virtually all missense variants in the BRCA1 RING domain, including variants that have not yet been observed in patients.

**Comment by melissacline** [comment]
[built_with_change_types.dictionary.xlsx](https://github.com/BRCAChallenge/brca-exchange/files/2912846/built_with_change_types.dictionary.xlsx)
Here is the data dictionary, checked over and ready to go.

**Comment by melissacline** [comment]
[README.txt](https://github.com/BRCAChallenge/brca-exchange/files/2912861/README.txt)
Here is an update for the top-level README for the data bundle

**Comment by melissacline** [comment]
Here is a better version, with the contact info from #1000 
[README.txt](https://github.com/BRCAChallenge/brca-exchange/files/2912878/README.txt)

**Comment by zfisch** [comment]
@melissacline I noticed the data dictionary is in .xlsx format -- would it make sense to use .tsv instead? I don't have excel, not sure if other users might run into a similar issue.

**Comment by melissacline** [comment]
Good point.  Here is the .tsv file (in the zip, github doesn't like tsv attachments) and an updated README.txt

[README.txt](https://github.com/BRCAChallenge/brca-exchange/files/2917061/README.txt)

[tmp.zip](https://github.com/BRCAChallenge/brca-exchange/files/2917081/tmp.zip)

**Comment by amycoffin** [comment]
@zfisch : This disclaimer has the most approval at the moment. I think it's fine that we might get more feedback on wording during beta:

Please note that these experiments may include variants that have not been observed in clinical contexts. Furthermore, functional assay results reflect laboratory models of genetic variation and disease. Thus, these results should not be used as a proxy for clinical variant interpretations.

**Comment by melissacline** [comment]
We didn't get any further feedback on yesterday's EGG call, so it looks like we can move forward with this one single item.

**Comment by zfisch** [comment]
@amycoffin is the final content for beta ready for a pr? I understand there’s a minor change we want to make before production deploy — or is beta currently ready to go?

**Comment by amycoffin** [comment]
@zfisch I just noticed an unwanted extra space in the first sentence of documentation:

 ...variants in functionally  critical domains...

Other than that, the disclaimer has been approved by Sam. I tried reworking it along Mary's feedback, but it wasn't really working. The information is there, and accurate, so any improvements can be made in future releases. 

tl;dr other than the random unwanted space, its ready IMO
